pennylane/ops/__init__.py

```diff
@@ -33,10 +33,6 @@
     conversion between the two conventions takes places automatically
     by the plugin device.
 
-
-Architecture-specific operations
---------------------------------
-
```

Comment:
As mentioned earlier, this works around a bug in sphinx, where section titles stop embedded TOCs from showing up in the sidebar.